easypost/postage_label.py

```diff
@@ -0,0 +1,5 @@
+from easypost.resource import AllResource, CreateResource
+
+
+class PostageLabel(AllResource, CreateResource):
```

Comment:
This should probably only inherit `Resource`

Reply:
good call